### PR TITLE
Add Swift Package Manager support

### PR DESCRIPTION
**Pull request by MortyMerr**
https://github.com/RxSwiftCommunity/RxSwiftExt/issues/228

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        .library(name: "RxSwiftExt", targets: ["RxSwiftExt"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ReactiveX/RxSwift.git", .upToNextMajor(from: "5.0.0")),
+    ],
+    targets: [
+        .target(name: "RxSwiftExt", dependencies: ["RxSwift", "RxCocoa"], path: "Source"),
+        .testTarget(name: "RxSwiftExtTests", dependencies: ["RxSwiftExt", "RxTest"], path: "Tests"),
+    ],
+    swiftLanguageVersions: [.v5]
```
Comment (freak4pc) [inline Package.swift]: Don't remember this off the top of my head but should this indicate .v5_1 as well? I think we also support Swift 4.2 here.

File: Source/RxCocoa/UIScrollView+reachedBottom.swift [hunk 2]
```diff
@@ -29,3 +30,5 @@ public extension Reactive where Base: UIScrollView {
         return ControlEvent(events: source)
     }
 }
+
```
Comment (freak4pc) [inline Source/RxCocoa/UIScrollView+reachedBottom.swift]: redundant empty line

File: Tests/RxCocoa/UIScrollView+reachedBottomTests.swift [hunk 3]
```diff
@@ -85,3 +86,5 @@ private extension UIScrollViewReachedBottomTests {
         }
     }
 }
+
```
Comment (freak4pc) [inline Tests/RxCocoa/UIScrollView+reachedBottomTests.swift]: redundant empty line

File: Tests/RxCocoa/UIViewPropertyAnimatorTests+Rx.swift [hunk 4]
```diff
@@ -59,3 +60,5 @@ class UIViewPropertyAnimatorTests: XCTestCase {
         XCTAssertEqual(animator.fractionComplete, 0.5)
     }
 }
+
```
Comment (freak4pc) [inline Tests/RxCocoa/UIViewPropertyAnimatorTests+Rx.swift]: redundant empty line

**Comment by SlashDevSlashGnoll** [comment]
Thanks for doing this.  This one repo is holding up a cascade of Rx-related ones we'd love to switch over to SPM!

**Comment by jessemx109** [comment]
can you create a new release with this change?

**Comment by MortyMerr** [comment]
@jessemx109 this is a question to Shai, as I understand we are going to accept a few more PRs for next release.

**Pull request merged**